ports/esp32s2/mpconfigport.mk

```diff
@@ -12,6 +12,8 @@ USB_SERIAL_NUMBER_LENGTH = 12
 # Longints can be implemented as mpz, as longlong, or not
 LONGINT_IMPL = MPZ
 
+CIRCUITPY_NEOPIXEL_WRITE = 1
```

Comment:
Please add this below in alphabetical order. That way it'll be easier to find.

Please also turn on CIRCUITPY_PIXELBUF. (It worked for me in my testing.)

Reply:
Was this meant to be alphabetical?

Reply:
@tannewt It's actually not supposed to need to be there at all - CIRCUITPY_NEOPIXEL_WRITE is supposed to be on by default but it'd bug out if I didn't put it here. You've reminded me I need to look into why that's happening.